src/Files.Uwp/UserControls/MultitaskingControl/HorizontalMultitaskingControl.xaml

```diff
@@ -50,6 +50,24 @@
                                 <FontIcon Glyph="&#xE8A7;" />
                             </MenuFlyoutItem.Icon>
                         </MenuFlyoutItem>
+                        <MenuFlyoutItem
+                            x:Uid="HorizontalMultitaskingControlPinTab"
+                            Click="{x:Bind vm:MainPageViewModel.PinTab}"
+                            Text="Pin tab"
+                            Visibility="{Binding IsPinned, Converter={StaticResource VisibilityInvertConverter}}">
+                            <MenuFlyoutItem.Icon>
+                                <FontIcon Glyph="&#xE718;" />
+                            </MenuFlyoutItem.Icon>
+                        </MenuFlyoutItem>
+                        <MenuFlyoutItem
+                            x:Uid="HorizontalMultitaskingControlUnPinTab"
+                            Click="{x:Bind vm:MainPageViewModel.UnpinTab}"
+                            Text="Unpin tab"
+                            Visibility="{Binding IsPinned, Converter={StaticResource BooleanToVisibilityConverter}}">
+                            <MenuFlyoutItem.Icon>
+                                <FontIcon Glyph="&#xE77A;" />
+                            </MenuFlyoutItem.Icon>
+                        </MenuFlyoutItem>
```

Comment:
```suggestion
                        <MenuFlyoutItem
                            Click="{x:Bind vm:MainPageViewModel.PinTab}"
                            Text="{helpers:ResourceString Name=PinTab}"
                            Visibility="{Binding IsPinned, Converter={StaticResource VisibilityInvertConverter}}">
                            <MenuFlyoutItem.Icon>
                                <FontIcon Glyph="&#xE718;" />
                            </MenuFlyoutItem.Icon>
                        </MenuFlyoutItem>
                        <MenuFlyoutItem
                            Click="{x:Bind vm:MainPageViewModel.UnpinTab}"
                            Text="{helpers:ResourceString Name=UnPinTab}"
                            Visibility="{Binding IsPinned, Converter={StaticResource BooleanToVisibilityConverter}}">
                            <MenuFlyoutItem.Icon>
                                <FontIcon Glyph="&#xE77A;" />
                            </MenuFlyoutItem.Icon>
                        </MenuFlyoutItem>
```

Reply:
Okay. I need to use helpers:ResourceString for multi languages. Thank you.

Reply:
The pull request is looking pretty good, still a couple of minor issues but I'll bring them up one at a time.